jwst/photom/ifu_mrs.py

```diff
@@ -0,0 +1,145 @@
+import logging
```

Comment:
Not a big deal, but given that this new module is exclusive to MIRI MRS, and not for all IFU modes, I would suggest renaming to `miri_mrs.py`.